app/src/main/java/org/covidwatch/android/presentation/MainActivity.kt

```diff
@@ -2,12 +2,27 @@ package org.covidwatch.android.presentation
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import org.covidwatch.android.BuildConfig
 import org.covidwatch.android.R
+import org.covidwatch.android.data.contactevent.ContactEventFetcher
+import org.koin.android.ext.android.inject
 
 class MainActivity : AppCompatActivity() {
 
+    private val contactEventFetcher: ContactEventFetcher by inject()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        if (BuildConfig.DEBUG) {
```

Comment:
Even though this is a test-related code we should avoid putting login into UI classes.
This could perfectly be used inside of `HomeViewModel` in the `init` and `onCleared` methods or we could introduce two new methods there such as `start`, `stop`